internal/api/api_client.go

```diff
@@ -21,18 +22,22 @@ func (e ClientError) Error() string {
 	return string(e)
 }
 
-func NewApiClient(httpClient HTTPClient) *ApiClient {
+func NewApiClient(httpClient HTTPClient, siriusUrl string, backendUrl string) (*ApiClient, error) {
```

Comment:
rename these to `siriusURL` and `backendURL` to match convention

Reply:
done